### PR TITLE
add pyvibms support

### PR DESCRIPTION
**Pull request by smutao**
use PyVibMS to visualize the vibrational analysis result by xtb

### CODE REVIEW

File: source/community/pyvibms.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
+PyVibMS
+=======
+
+To visualize the vibrational modes calculated by xtb (via ``--hess`` or ``--ohess``), find the ``g98.out`` file which collects the vibrational modes and frequencies.
+
+Lauch PyVibMS in a new PyMOL window, load the  ``g98.out`` file by changing the file type into **Output File (*.out)**. Change the program type from **XYZ** to **xtb (g98.out file)**, check the **Has. Vib. Info.** box, click **Load** button.
+
+Then vibrational modes can be visualized as following
+
+.. image:: https://raw.githubusercontent.com/smutao/PyVibMS/master/examples/xtb-640/xtb-pyvibms-shot.png
```
Comment (awvwgk) [inline source/community/pyvibms.rst:10]: Can we pin this to a commit hash to ensure the link keeps working?

Comment (awvwgk) [reply source/community/pyvibms.rst:10]:
```suggestion
.. image:: https://raw.githubusercontent.com/smutao/PyVibMS/83e503c0abf3f5f93cc667d28f67a433073c6757/examples/xtb-640/xtb-pyvibms-shot.png
```

Comment (smutao) [reply source/community/pyvibms.rst:10]: Sure, no problem. Thanks.

**Review by awvwgk — APPROVED** [review]
Thanks for sharing, great work.

**Pull request merged**